python/docs/hashtable_repeated_word/README.md

```diff
@@ -0,0 +1,41 @@
+# Challenge Summary
+Find repeated words
+
+## Whiteboard Process
+[whiteboard](python/docs/hashtable_repeated_word/whiteboard.jpeg)
```

Comment:
The linked image is probably not what you meant to link. The link is for the sum_matrix_rows() whiteboard.